### PR TITLE
1. Replaced \n with System.lineseperator()

### PR DESCRIPTION
**Pull request by appeltone**
1. Replaced \n with System.lineseperator()
2. Removed un-needed inheritance from Calculator

### CODE REVIEW

**Review by Dgotlieb — APPROVED** [review]
Looks great

File: src/calculator/UtilsClass.java [hunk 1]
```diff
@@ -1,28 +1,67 @@
...
+            e.printStackTrace();
+        }
+
+    }
+
+    public static Connection getDbConnection() {
+        Connection conn = null;
+        Properties connectionProps = new Properties();
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
```
Comment (Dgotlieb) [inline src/calculator/UtilsClass.java:37]: This is deprecated and there is no need for it since java 7

File: src/calculator/UtilsClass.java [hunk 2]
```diff
@@ -61,6 +100,17 @@ public static void printResult(double answer) {
             Date date = new Date(System.currentTimeMillis());
 
 
+            //print to results column in my remotemysql db
+            Connection con = getDbConnection();
+            try {
+                Statement st = con.createStatement();
+                st.execute("INSERT INTO `calculator_db` (`" + RESULT_COLUMN + "`) VALUES ('" + answer + "')");
```
Comment (Dgotlieb) [inline src/calculator/UtilsClass.java]: Great job!
Please replace with prepared statements:
https://docs.oracle.com/javase/tutorial/jdbc/basics/prepared.html

**Review by Dgotlieb — CHANGES_REQUESTED** [review]
This is great!
Please convert your project to Maven based and include all dependencies (including mysql-connector to pom.xml)

File: src/calculator/UtilsClass.java (hunk 1 above)
Comment (appeltone) [reply src/calculator/UtilsClass.java:37]: oh, i understand why i got the warnings now, thanks

Comment (appeltone) [reply src/calculator/UtilsClass.java:37]: you mean Class.forName("com.mysql.jdbc.Driver"); right ?

File: src/calculator/UtilsClass.java (hunk 2 above)
Comment (appeltone) [reply src/calculator/UtilsClass.java]: Done

File: src/calculator/UtilsClass.java (hunk 1 above)
Comment (Dgotlieb) [reply src/calculator/UtilsClass.java:37]: Yes

File: src/calculator/UtilsClass.java (hunk 2 above)
Comment (Dgotlieb) [reply src/calculator/UtilsClass.java]: I don't see the prepared statement object.. did you forget to commit it?

**Review by Dgotlieb — COMMENTED** [review]
Missing prepared statement

**Comment by Dgotlieb** [comment]
Please commit your pom.xml file

**Review by Dgotlieb — APPROVED** [review]
Looks great!
tiny change... please change target from java 7 to at least 8 and merge

**Pull request merged**